src/outputs/history.cpp

```diff
@@ -275,8 +285,39 @@ void HistoryOutput::WriteOutputFile(Mesh *pm, ParameterInput *pin, bool flag) {
       for (int n=0; n<NSCALARS; n++) {
         std::fprintf(pfile,"[%d]=%d-scalar    ", iout++, n);
       }
+      if (NR_RADIATION_ENABLED || IM_RADIATION_ENABLED) {
+        if (pm->my_blocks(0)->pnrrad->nfreq == 1) {
+          std::fprintf(pfile,"[%d]=Er    ", iout++);
+          std::fprintf(pfile,"[%d]=Fr1    ", iout++);
+          std::fprintf(pfile,"[%d]=Fr2    ", iout++);
+          std::fprintf(pfile,"[%d]=Fr3    ", iout++);
+          std::fprintf(pfile,"[%d]=Er0    ", iout++);
+          std::fprintf(pfile,"[%d]=Fr10    ", iout++);
+          std::fprintf(pfile,"[%d]=Fr20    ", iout++);
+          std::fprintf(pfile,"[%d]=Fr30    ", iout++);
+          std::fprintf(pfile,"[%d]=Pr11    ", iout++);
+          std::fprintf(pfile,"[%d]=Pr12    ", iout++);
+          std::fprintf(pfile,"[%d]=Pr13    ", iout++);
+          std::fprintf(pfile,"[%d]=Pr22    ", iout++);
+          std::fprintf(pfile,"[%d]=Pr23    ", iout++);
+          std::fprintf(pfile,"[%d]=Pr33    ", iout++);
+        } else {
+          for (int ifr=0; ifr<pm->my_blocks(0)->pnrrad->nfreq; ++ifr) {
+            std::fprintf(pfile,"[%d]=Er   ", iout++);
+            std::fprintf(pfile,"[%d]=Fr1   ", iout++);
+            std::fprintf(pfile,"[%d]=Fr2   ", iout++);
+            std::fprintf(pfile,"[%d]=Fr3   ", iout++);
+          }
+        }
+      }
+      if (CR_ENABLED) {
+        std::fprintf(pfile,"[%d]=Ec    ", iout++);
+        std::fprintf(pfile,"[%d]=Fc1    ", iout++);
+        std::fprintf(pfile,"[%d]=Fc2    ", iout++);
+        std::fprintf(pfile,"[%d]=Fc3    ", iout++);
+      }
       for (int n=0; n<pm->nuser_history_output_; n++)
-        std::fprintf(pfile,"[%d]=%-7s ", iout++,
+        std::fprintf(pfile,"[%d]=%-8s", iout++,
```

Comment:
why did you change the number of characters?

Reply:
> why did you change the number of characters?

The history output file has been changed to be consistent with master version.
@felker Is NGRAV_VARIABLES used anywhere? This is defined in defs.hpp.in